webpack/assets/javascripts/react_app/common/urlHelpers.js

```diff
@@ -69,3 +70,23 @@ export const changeQuery = (newQuery, navigateTo, uri = getURI()) => {
   if (navigateTo) navigateTo(uri.toString());
   else window.Turbolinks.visit(uri.toString());
 };
+
+export const foremanshortVersion = () =>
+  packageJson.version
+    .split('.')
+    .slice(0, 2)
+    .join('.');
+
+export const documentationURL = (section = '', options = {}) => {
+  const shortVersion = foremanshortVersion();
+  const rootUrl =
+    options.rootUrl ||
+    `https://theforeman.org/manuals/${shortVersion}/index.html#`;
+  if (!section) return rootUrl;
```

Comment:
Technically this doesn't really matter unless section is null instead of an empty string.

Reply:
if section = null it will create a string with the word null `https://theforeman.org/manuals/${shortVersion}/index.html#null`